cluster-autoscaler/cloudprovider/gce/templates_test.go

```diff
@@ -98,6 +103,23 @@ func TestBuildNodeFromTemplateSetsResources(t *testing.T) {
 			kubeReserved:   false,
 			expectedErr:    false,
 		},
+		{
+			scenario: "BLOCK_EPH_STORAGE_BOOT_DISK in kube-env",
```

Comment:
can we also add a case where BLOCK_EPH_STORAGE_BOOT_DISK is in kube-env, but with a value of 'false' or some other non-true value?

Reply:
Done